docs/content/users/quickstart.md

```diff
@@ -318,6 +318,15 @@ In the web browser, log into your account using `admin` and `publish`.
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Joomla
+
+```bash
+mkdir my-joomla-site && cd my-joomla-site
+ddev config --project-type=php
+# Download the latest version from https://downloads.joomla.org/latest and unarchive everything into the empty my-joomla-site directory
```

Comment:
Why not show this step? Can it be done with curl?

I think it might be better to do the download, have it create the directory? and then do the config?

Reply:
but as already said on slack none of the links available provides a direct link to the latest version joomla like backdrop provides for example. all those links are only links to pages that contain several links to the latest version in the different compression formats. but none of those is directly downloadable via curl without jumping through any hoops :(

Reply:
Use this:

```
tag=$(curl -L "https://api.github.com/repos/joomla/joomla-cms/releases/latest" | docker run -i --rm ddev/ddev-utilities jq -r .tag_name) && curl -L "https://github.com/joomla/joomla-cms/releases/download/$tag/Joomla_$tag-Stable-Full_Package.zip" -o joomla.zip
unzip ./joomla.zip && rm joomla.zip
```